From 233a712feeaf3c89b4739a66cc533a3bf0bf8af2 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Fri, 1 Jan 2021 15:43:56 +0100
Subject: [PATCH 14/21] Fix an undefined reference when TARGET

The recipe was probably adapted from $* to $TARGET, but one $* was not
replaced.  Moreover, DEB_TARGET_MULTIARCH was defined but unused yet.
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 9c0e136..7e155ae 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1526,7 +1526,7 @@ ifneq (,$(TARGET))
 
 	rm -f debian/substvars
 	dpkg-shlibdeps $(d_cross)/$(PF)/bin/* \
-	  -ldebian/tmp/$(PF)/lib/$(call _multiarch,$*) -l$(d_lib)/$(PF)/lib/$(call _multiarch,$*)
+	  -ldebian/tmp/$(PF)/lib/$(DEB_TARGET_MULTIARCH) -l$(d_lib)/$(PF)/lib/$(DEB_TARGET_MULTIARCH)
 	dpkg-gencontrol -P$(d_cross) -p$(p_cross) \
 		-VBuilt-Using="$(shell dpkg-query -f '$${source:Package} (= $${source:Version}), ' -W binutils-source)"
 	find $(d_cross) -depth -newermt '$(BUILD_DATE)' -print0 | \
-- 
2.30.1

